generator/internal/rust/annotate_test.go

```diff
@@ -325,6 +325,57 @@ func TestEnumAnnotations(t *testing.T) {
 	}
 }
 
+func TestDuplicateEnumValueAnnotations(t *testing.T) {
+	// Verify we can handle values that are not in SCREAMING_SNAKE_CASE style.
+	v0 := &api.EnumValue{
+		Name:   "full",
+		ID:     ".test.v1.TestEnum.full",
+		Number: 1,
+	}
+	v1 := &api.EnumValue{
+		Name:   "FULL",
+		ID:     ".test.v1.TestEnum.FULL",
+		Number: 1,
+	}
+	v2 := &api.EnumValue{
+		Name:   "partial",
+		ID:     ".test.v1.TestEnum.partial",
+		Number: 2,
+	}
+	// This does not happen in practice, but we want to verify the code can
```

Comment:
This gave me pause. Maybe the generator should panic so we can go yell at the service team?

If we do receive the value 3, I guess we will just treat it as an UNKNOWN_VALUE:3 (or whatever). Which should be fine.

Reply:
> This gave me pause. Maybe the generator should panic so we can go yell at the service team?

I thought about that, but then we would stuck: `sidekick update` fails for all services, not just the offending one. We would need to hack something.

> If we do receive the value 3, I guess we will just treat it as an UNKNOWN_VALUE:3 (or whatever). Which should be fine.

Exactly. It is very unlikely to happen (https://google.aip.dev/126 says enum values **must** be SCREAMING_SNAKE_CASE, the deviations are service bugs).  And if it does happen we still can handle the incoming enums.